continuous_integration/recipe/meta.yaml

```
@@ -41,6 +43,8 @@ requirements:
     - prompt-toolkit >=3.0.8
     - pygments >=2.7.1
     - tabulate
+  run_constrained:
+    - __glibc >=2.32  # [aarch64]
```

Suggesting adding the same comment that was added for wheels here

```suggestion
  run_constrained:
    # FIXME: aarch64 builds require glibc>=2.32 to prevent TLS allocation error when importing;
    # is there any way we could modify the TLS model of its shared object to avoid this?
    # https://bugzilla.redhat.com/show_bug.cgi?id=1722181
    # xref: https://github.com/dask-contrib/dask-sql/issues/1169
    - __glibc >=2.32  # [aarch64]
```